Modelica/Resources/C-Sources/snprintf.c

```diff
@@ -522,7 +522,7 @@ static UINTMAX_T cast(LDOUBLE);
 static UINTMAX_T myround(LDOUBLE);
 static LDOUBLE mypow10(int);
 
-//extern int errno;
+/*extern int errno;*/
```

Comment:
Good, especially as Visual Studio nowadays have their own definition of it as I recall.